cmd/promtool/main.go

```diff
@@ -557,7 +557,7 @@ func CheckServerStatus(serverURL *url.URL, checkEndpoint string, roundTripper ht
 		return err
 	}
 
-	request, err := http.NewRequest(http.MethodGet, config.Address, nil)
+	request, err := http.NewRequest(http.MethodGet, config.Address, http.NoBody)
```

Comment:
Just wondering.. why is this even needed. I see "major anti-pattern" (https://deepsource.com/directory/go/issues/GO-R4001), but I don't see negative of passing nil, really, other than readability. NewRequests handles nil 🙈 WTH this is needed